internal/driver/glfw/window.go

```diff
@@ -52,11 +52,12 @@ type window struct {
 	decorate   bool
 	fixedSize  bool
 
-	cursor   *glfw.Cursor
-	canvas   *glCanvas
-	title    string
-	icon     fyne.Resource
-	mainmenu *fyne.MainMenu
+	cursor    desktop.Cursor
+	rawCursor *glfw.Cursor // rawCursor is only filled for custom cursors in order to be destroyed when becoming un-needed
```

Comment:
I’d prefer a name that delivers the intent of this property. That would save the comment and also make the code in `mouseMoved` easier to understand.
AFAICS a fairly complete name would be `customGLFWCursorToBeFreed` but this is too long, of course.

I can see 2 (and a half) things that make this “cursor” different from the `cursor` property:
1. it is a GLFW cursor (I guess `raw` should mediate this)
2a. it is custom
2b. and should therefore be freed after use

Probably, we can focus on 2. Maybe `customCursor` is already enough?
Or `customRawCursor` which would contain a hint on 1, too?